Add a clarification that workflow end_to_end latency covers only one workflow run (#1013)
temporalio/documentation: docs/reference/metrics.md

### `workflow_endtoend_latency`

> Total Workflow Execution time from schedule to completion for a single Workflow Run (A retried Workflow Execution is a separate Run). 

Reviewer: ```suggestion
Total Workflow Execution time from schedule to completion for a single Workflow Run. (A retried Workflow Execution is a separate Run.)
```

> Total Workflow Execution time from schedule to completion. Covers one Workflow Run. Workflow Retries are represented as different Runs. 

Reviewer 2: ```suggestion
Total Workflow Execution time from schedule to completion for a single Workflow Run (A retried Workflow Execution is a separate Run). 
```
